src/mpfao/tdympfao.c

```diff
@@ -470,6 +518,14 @@ PetscErrorCode TDyMPFAOInitialize(TDy tdy) {
   if (tdy->ops->computepermeability) {ierr = SetPermeabilityFromFunction(tdy); CHKERRQ(ierr);}
   if (tdy->mode == TH){
     if (tdy->ops->computethermalconductivity) {ierr = SetThermalConductivityFromFunction(tdy); CHKERRQ(ierr);}
+    if (tdy->ops->computesoildensity) {
+      printf("SetSoilDensityFromFunction\n");
```

Comment:
Do you want to leave this and the other `printf` below in here? Or are these just for debugging?

Reply:
done